charts/memgraph-high-availability/README.md

```diff
@@ -59,12 +58,18 @@ The following table lists the configurable parameters of the Memgraph chart and
 | `memgraph.coordinators.volumeClaim.storagePVCSize` | Size of the storage PVC for coordinators                                                         | `1Gi`                                   |
 | `memgraph.coordinators.volumeClaim.logPVC`  | Enable log PVC for coordinators                                                                     | `false`                                 |
 | `memgraph.coordinators.volumeClaim.logPVCSize` | Size of the log PVC for coordinators                                                              | `256Mi`                                 |
-| `memgraph.affinity.enabled`                 | Enables affinity so each instance is deployed to unique node                                        | `true`                                 |
 | `memgraph.externalAccess.serviceType`       | NodePort or LoadBalancer. Use LoadBalancer for Cloud production deployment and NodePort for local testing | `LoadBalancer`                    |
 | `memgraph.ports.boltPort`                   | Bolt port used on coordinator and data instances.                                                   | `7687`                                  |
 | `memgraph.ports.managementPort`             | Management port used on coordinator and data instances.                                             | `10000`                                 |
 | `memgraph.ports.replicationPort`            | Replication port used on data instances.                                                            | `20000`                                 |
 | `memgraph.ports.coordinatorPort`            | Coordinator port used on coordinators.                                                              | `12000`                                 |
+| `memgraph.affinity.enabled`                        | Enables affinity so each instance is deployed to unique node                                        | `true`                     |
```

Comment:
if I am not wrong, we don't need affinity.enabled anymore.

Reply:
Fixed.